example-get-started-experiments/code/.github/workflows/deploy-model.yml

```diff
@@ -0,0 +1,58 @@
+name: Deploy model
+
+on:
+  push:
+    # When a new version is registered in Studio Model Registry
+    tags:
```

Comment:
Did not use gto action because it did simplify / add much value for the deployment pattern used